src/jats/importer/jats-body-transformations.ts

```diff
@@ -24,6 +24,23 @@ const removeNodeFromParent = (node: Element) =>
 const capitalizeFirstLetter = (str: string) =>
   str.charAt(0).toUpperCase() + str.slice(1)
 
+const createSectionContainer = (
+  type: string,
+  createElement: (tagName: string) => HTMLElement
+) => {
+  const sectionContainer = createElement('sec')
+  const category = chooseSectionCategoryByType(type)
+  sectionContainer.setAttribute(
+    'sec-type',
+    category ? chooseSecType(category) : ''
+  )
+
+  const title = createElement('title')
+  title.textContent = category ? chooseSecType(category) : ''
```

Comment:
Can we use the section category title here?

Reply:
A couple of questions on this:
Do you want me to bring them from manuscripts-data or or use a switch statement similar to [chooseSectionCategoryFromTitle](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/1556eb5b3613409da630626c3ed12de60c54ef0f/src/transformer/section-category.ts#L306)  or something else?
In manuscripts-data, the titles have been left empty for the new sections. I think its better If I add the following titles:
- backmatter -> backmatter
- body -> body
- abstracts-> abstracts

I'm not sure about capitalizing the first letter of the title, there are some inconsistencies in the data
Should I proceed with this change ?

Reply:
You should use whatever is in manuscripts-data. It's fine if it's empty for now (I'm assuming that it will not break with an empty title)